### PR TITLE
Config changes to deploy to Heroku

### PR DESCRIPTION
**Pull request by mermop**
it's up on https://snek-farm.herokuapp.com/ now - I think there are some other things that need changing to get it running safely in production.

- Commits `main.js` again for the moment - maybe we can do a build pipeline later
- Config to use Postgres - but we can still use sqlite locally
- You'll need to copy `env.example` to `env`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,7 +25,6 @@ wheels/
 *.egg
 MANIFEST
 node_modules/
-frontend/static/frontend/main.js
```
Comment (oheyjesse) [inline .gitignore:28]: Mmm. Makes sense for a `build from github` style deploy system. Maybe we can at least rename the files / put them in some kind of `build` directory to make it more obvious what's going on here

File: requirements.txt [hunk 2]
```diff
@@ -7,4 +7,9 @@ pycodestyle==2.5.0
 pyflakes==2.1.1
 pytz==2019.2
 sqlparse==0.3.0
-djangorestframework~=3.10.2	
+djangorestframework~=3.10.2
+gunicorn==19.9.0
+whitenoise==4.1.3
+psycopg2-binary==2.8.3
+python-dotenv==0.10.3
+dj-database-url==0.5.0
```
Comment (oheyjesse) [inline requirements.txt:15]: I read this as `DJ DATABASE IN THE HOUSSSSEEEEE` *bew bew bewwwwww*

File: snekdancer/settings.py [hunk 3]
```diff
@@ -115,9 +119,27 @@
 
 USE_TZ = True
 
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
-# Static files (CSS, JavaScript, Images)
-# https://docs.djangoproject.com/en/2.2/howto/static-files/
+# this line is already in your settings.py
```
Comment (oheyjesse) [inline snekdancer/settings.py]: Is this the right comment? Should it refer more to something else?

**Review by oheyjesse — APPROVED** [review]
Looks great, merrin! One comment about a comment, but it's non-blocking. :D Thanks for putting in the work 👍

File: requirements.txt (hunk 2 above)
Comment (libbyberrie) [reply requirements.txt:15]: 📣 📣

File: .gitignore (hunk 1 above)
Comment (libbyberrie) [reply .gitignore:28]: I agree with Drew here, though maybe for another PR?

**Review by libbyberrie — APPROVED** [review]
Looks great to me! Thanks for being so wonderful and efficient 💪

**Comment by oheyjesse** [comment]
> Looks great to me! Thanks for being so wonderful and efficient 💪

I'm learning so much just by basking in the Merrin Glow

File: snekdancer/settings.py (hunk 3 above)
Comment (mermop) [reply snekdancer/settings.py]: Lol oops that is copied from a walkthrough

File: .gitignore (hunk 1 above)
Comment (mermop) [reply .gitignore:28]: Yeah I reckon

**Pull request merged**